extensions/modules/file/src/main/java/org/exist/xquery/modules/file/Sync.java

```diff
@@ -88,15 +93,21 @@ public Sequence eval(final Sequence[] args, final Sequence contextSequence) thro
 		}
 		
 		final String collectionPath = args[0].getStringValue();
+
+		final String target = args[1].getStringValue();
+		Path targetDir = FileModuleHelper.getFile(target);
+
 		Date startDate = null;
 		if (args[2].hasOne()) {
 			DateTimeValue dtv = (DateTimeValue) args[2].itemAt(0);
```

Comment:
Could `dv` and `bv` be `final`?

Reply:
Done, but why do this for block-scoped vars, that go out of scope two lines further?

Reply:
As a code style thing - we just try to prefer that all variables are `final` unless they can't be for some logical reason.